### ISSUE TITLE
Support buf.build registry

### ISSUE THREAD

**Issue by birdayz** [birdayz/kaf#106]
They will implement a kind of schema registry for proto. I'm convinced buf will get significant traction, given that even the makers of prototool are endorsing it. This can be built on top of kaf's protobuf support, where we fetch descriptors from the filesystem.

**Comment by thesiti92** [comment]
yeah this would be incredibly helpful, we use buf at my company and i have been having trouble setting it up for kaf to parse the messages locally.
